src/graph/test/SchemaTest.cpp

```diff
@@ -725,6 +725,44 @@ TEST_F(SchemaTest, metaCommunication) {
         }
         ASSERT_TRUE(verifyResult(resp, expected));
     }
+    // Tag with negative default value
+    {
+        cpp2::ExecutionResponse resp;
+        const std::string cmd =
+            "CREATE TAG default_tag_neg(id int DEFAULT -10, height double DEFAULT -176.0)";
+        auto code = client->execute(cmd, resp);
+        EXPECT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
+    }
+    // Tag with expression default value
+    {
+        cpp2::ExecutionResponse resp;
+        const std::string cmd = "CREATE TAG default_tag_expr"
+            "(id int DEFAULT 3/2*4-5, "  // arithmetic
+            "male bool DEFAULT 3 > 2, "  // relation
+            "height double DEFAULT abs(-176.0), "  // built-in function
+            "adult bool DEFAULT true && false)";  // logic
+        auto code = client->execute(cmd, resp);
+        EXPECT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
+    }
+    // Edge with negative default value
+    {
+        cpp2::ExecutionResponse resp;
+        const std::string cmd =
+            "CREATE EDGE default_edge_neg(id int DEFAULT -10, height double DEFAULT -176.0)";
+        auto code = client->execute(cmd, resp);
+        EXPECT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
+    }
+    // Edge with expression default value
+    {
+        cpp2::ExecutionResponse resp;
+        const std::string cmd = "CREATE EDGE default_edge_expr"
+            "(id int DEFAULT 3/2*4-5, "  // arithmetic
+            "male bool DEFAULT 3 > 2, "  // relation
+            "height double DEFAULT abs(-176.0), "  // built-in function
+            "adult bool DEFAULT true && false)";  // logic
+        auto code = client->execute(cmd, resp);
+        EXPECT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
```

Comment:
better to check the result, add `show create tag` to check

Reply:
Yes, I plan to add checking if https://github.com/vesoft-inc/nebula/pull/1730 merged.